pkg/apply/taskrunner/task.go

```diff
@@ -131,13 +131,13 @@ func (w *WaitTask) Start(taskContext *TaskContext) {
 	}()
 }
 
-func (w *WaitTask) sendEvent(taskContext *TaskContext, id object.ObjMetadata, op event.WaitEventOperation) {
+func (w *WaitTask) sendEvent(taskContext *TaskContext, id object.ObjMetadata, op event.WaitEventStatus) {
```

Comment:
nit: should the parameter be changed to `status` rather than `op`?

Reply:
good catch. i caught all the "operation" ones but not the "op" ones. will fix.